TADashboard/src/main/java/base_pages/GeneralSettings_BasePage.java

```diff
@@ -0,0 +1,23 @@
+package base_pages;
+
+import com.logigear.control.common.imp.Button;
+import com.logigear.control.common.imp.Label;
+
+public class GeneralSettings_BasePage extends BasePage {
```

Comment:
what is the difference between GeneralSetting and BasePage? and what naming convention of a class is this?

Reply:
this naming is not acceptable